<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="keywordInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze Tweets</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>801</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tweetsInput">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>110</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMediaMiner">
    <property name="title">
     <string>MediaMiner</string>
    </property>
   </widget>
   <addaction name="menuMediaMiner"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
